webui/react/src/pages/FlatRuns/columns.ts

```diff
@@ -99,6 +104,13 @@ function getCellStateFromExperimentState(expState: RunState) {
   }
 }
 
+const EMPTY_CELL: GridCell = {
+  allowOverlay: false,
+  copyData: '-',
+  data: { kind: TEXT_CELL },
+  kind: GridCellKind.Custom,
```

Comment:
nit: why should this be a custom cell type instead of just text?

Reply:
That causes type error with `GridCellKind.TEXT`.

Reply:
It's merged now so I guess it's too late, but this doesn't give me a type error:
```
const EMPTY_CELL: GridCell = {
  allowOverlay: false,
  data: '-',
  kind: GridCellKind.Text,
};
```

Reply:
<img width="972" alt="image" src="https://github.com/user-attachments/assets/c41fc7f3-7a82-4f88-9b82-26fc39f2f40d">

looks like this works. i think we can change it with searches table changes

```ts
const EMPTY_CELL: GridCell = {
  allowOverlay: false,
  data: '-',
  displayData: '-',
  kind: GridCellKind.Text,
} as const;
```